config.py

```diff
@@ -3165,10 +3165,11 @@ def _find_bad_channels(cfg, raw, subject, session, task, run) -> None:
 
     # Filter the data manually before passing it to find_bad_channels_maxwell()
     # This reduces memory usage!
```

Comment:
Is this comment still relevant now? Can we have raw without preload up to this point to avoid memory issues?